src/modules/editingTask.js

```diff
@@ -0,0 +1,40 @@
+import modifyTask from './modifyTask.js';
+
+function editingTask(event) {
+  const selectedElementClass = `.${event.path[2].classList[0]}`;
```

Comment:
- [ ] After clicking on a task to edit, event.path[2] is not properly resolving (the event object has no path property as seen below)
![image](https://user-images.githubusercontent.com/61377322/166099348-ea9a70dc-f8f1-401c-a439-f468ae498a5c.png)


- hint: kindly check `event.target` to grab the target element for the event

Reply:
@bini-i How can I reproduce this error?

Reply:
Hi @wandergithub. You can reproduce the error by trying to edit a given task description with your console open. As soon as you focus on task description, editingTask method is called and you will be able to see the error in your console.

Reply:
Sorry, but for me, it works well.  However, I'll try to get the element by `event.target` instead.

Reply:
I am sorry @wandergithub, I tested you project only on firefox, I should have tested it on multiple browsers. 

The path property of event is not supported on firefox as explained [here](https://stackoverflow.com/a/39245638).